### ISSUE TITLE
Branches are confusing

### ISSUE THREAD

**Issue by brettviren** [DUNE-DAQ/ers#2]
This is dune-daq's fork.  We should use normal branches for our changes not `dune/devel`.  If we need to track ATLAS work we can do that in a special branch.

**Comment by alessandrothea** [comment]
Sure, let us get appfwk v1.0 out and then we can come up for a branch scheme convention for forked repos.

**Comment by brettviren** [comment]
At least for now mark `dune/devel` as "default" in the settings?

**Comment by alessandrothea** [comment]
Good idea. That and changing the name to dune/master shall make things a bit clearer.

**Comment by alessandrothea** [comment]
Done.
